.github/workflows/CI.yml

```diff
@@ -58,11 +58,13 @@ jobs:
           path: site/
 
       # danger for PR builds
-      - if: github.event_name == 'pull_request' && github.event.pull_request.base.repo.id == github.event.pull_request.head.repo.id
-        run: "yarn danger ci"
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          PR_DEPLOY_URL_ROOT: ${{ steps.deploy.outputs.static_web_app_url }}
+      - name: "Run Danger"
+        run: |
+          # Exposing this token is safe because the user of it has no other public repositories
+          # and has no permission to modify this repository. See DefinitelyTyped #62638 for the discussion.
+          TOKEN='ghp_i5wtj1l2AbpFv3OU96w6R'
```

Comment:
Even after reading through the comment thread at https://github.com/Definitelytyped/DefinitelyTyped/issues/62638, I'm still not a fan of using an unprotected token. Making this token public means anyone could use it to attempt to DDoS GitHub and have it falsely attributed to the DangerBotOSS account, or DoS Danger by artificially using up its rate limit. If making this a secret is not viable for usability reasons, are there any other mechanisms that could be employed to avoid exposing the token?

Reply:
I'm not really sure why this needs a token at all, either; isn't this just a CI check that can fail and print out the files that were forgotten? a la https://github.com/microsoft/TypeScript/blob/main/.github/workflows/ci.yml#L239

I guess because this tries to print a fancy comment?

Reply:
It's because you need a token to make a comment on an issue, yes!

![image](https://github.com/microsoft/TypeScript-Website/assets/49038/3239c214-8675-465a-a780-700eb76f79c5)

Yep, it's a comment because no-one would read a non-failing CI build for things like warnings

I did explore having a central github app https://github.com/danger/danger-js/pull/1126 but it requires giving too much github access to the bot IMO, and I didn't want to centralize that many people's tokens on my spare time